dts/bindings/memory-controllers/nuvoton,npcx-bbram.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2021 Google Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nuvoton, NPCX Battery Backed RAM node
+
+compatible: "nuvoton,npcx-bbram"
+
+include: base.yaml
+
+properties:
+    status:
+        required: true
```

Comment:
Can we remove `required: true` here or add `status = "disabled";` to npcx.dtsi bbram node.